### ISSUE TITLE
pcd_to_pointcloud should support compression

### ISSUE THREAD

**Issue by LucidOne** [ros-perception/perception_pcl#81]
Need to add inverse of #76 to support reading compressed PCD files

**Comment by AustinDeric** [comment]
I agree with this and i would like to see more options than just the string to the filename.  For example, i would like this function to take a file object or an http url.

**Comment by OAkyildiz** [comment]
Old issue but what are the limitations around the PCD file? Would clustering and publishing by cluster make it easier to load bigger files?
